### DISCUSSION TITLE
how to install scala 3?

### DISCUSSION THREAD

**Discussion by xhudik** [coursier/coursier#1968] [General]
Hi guys,
a newbie question: i went through a lot of documentation about scala3/dotty - how to install it but I didnt find any real commands how to do it. While: https://docs.scala-lang.org/scala3/getting-started.html mentions `cs` as a standard tool - after installation I ended up with scala 2 on my system.

When I checked your documentation - i learned how to install various version of java but nothing about scala versioning. Can you help me here - it might well be that I missed something.

Questions:
1. how to install scala 3 via `cs`
2. how to install various scala versions via `cs` (can they live in parallel)?

thank you, Tomas

**Comment by alexarchambault** [comment]
Hi,
So when one "installs" Scala, we don't really install the compiler, but rather build tool launchers (typically sbt, sometimes mill). `cs setup`, mentioned in your link, can install those. `cs setup` also installs a `scala` launcher, but it's not used when you run `sbt` for example, and doesn't prevent using any other Scala version from sbt or mill.

To get help about "installing Scala", I'd recommend asking on https://users.scala-lang.org or maybe https://gitter.im/scala/scala, which have a larger audience than the coursier issues or discussion.
